QSG-pack/src/main/resources/QSG/bin/02/response_unformatted.xml

```diff
@@ -0,0 +1 @@
+<?xml version='1.0' encoding='UTF-8'?><soapenv:Envelope xmlns:soapenv="http://schemas.xmlsoap.org/soap/envelope/"><soapenv:Body><ns:getApplicationResponse xmlns:ns="http://org.apache.axis2/xsd"><ns:return xmlns:ax2140="http://model.common.application.identity.carbon.wso2.org/xsd" xmlns:ax2138="http://common.application.identity.carbon.wso2.org/xsd" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:type="ax2140:ServiceProvider"><ax2140:applicationID>3</ax2140:applicationID><ax2140:applicationName>swift</ax2140:applicationName><ax2140:claimConfig xsi:type="ax2140:ClaimConfig"><ax2140:alwaysSendMappedLocalSubjectId>false</ax2140:alwaysSendMappedLocalSubjectId><ax2140:localClaimDialect>true</ax2140:localClaimDialect><ax2140:roleClaimURI xsi:nil="true"/><ax2140:userClaimURI xsi:nil="true"/></ax2140:claimConfig><ax2140:description>A Quick Start Guide</ax2140:description><ax2140:inboundAuthenticationConfig xsi:type="ax2140:InboundAuthenticationConfig"/><ax2140:inboundProvisioningConfig xsi:type="ax2140:InboundProvisioningConfig"><ax2140:dumbMode>false</ax2140:dumbMode><ax2140:provisioningEnabled>false</ax2140:provisioningEnabled><ax2140:provisioningUserStore xsi:nil="true"/></ax2140:inboundProvisioningConfig><ax2140:localAndOutBoundAuthenticationConfig xsi:type="ax2140:LocalAndOutboundAuthenticationConfig"><ax2140:alwaysSendBackAuthenticatedListOfIdPs>false</ax2140:alwaysSendBackAuthenticatedListOfIdPs><ax2140:authenticationStepForAttributes xsi:nil="true"/><ax2140:authenticationStepForSubject xsi:nil="true"/><ax2140:authenticationType>default</ax2140:authenticationType><ax2140:enableAuthorization>false</ax2140:enableAuthorization><ax2140:subjectClaimUri xsi:nil="true"/><ax2140:useTenantDomainInLocalSubjectIdentifier>false</ax2140:useTenantDomainInLocalSubjectIdentifier><ax2140:useUserstoreDomainInLocalSubjectIdentifier>false</ax2140:useUserstoreDomainInLocalSubjectIdentifier></ax2140:localAndOutBoundAuthenticationConfig><ax2140:outboundProvisioningConfig xsi:type="ax2140:OutboundProvisioningConfig"><ax2140:provisionByRoleList xsi:nil="true"/></ax2140:outboundProvisioningConfig><ax2140:owner xsi:type="ax2140:User"><ax2140:tenantDomain>carbon.super</ax2140:tenantDomain><ax2140:userName>cameron</ax2140:userName><ax2140:userStoreDomain>PRIMARY</ax2140:userStoreDomain></ax2140:owner><ax2140:permissionAndRoleConfig xsi:type="ax2140:PermissionsAndRoleConfig"/><ax2140:saasApp>false</ax2140:saasApp></ns:return></ns:getApplicationResponse></soapenv:Body></soapenv:Envelope>
```

Comment:
Add a new line to the file

Reply:
This is file contains a SOAP response. Therefore we can't add new lines manually.